templates/graffitiwall.html

```diff
@@ -95,8 +120,12 @@ <h2 class="h6">
     </div>
     <br />
     <div class="row">
-      <div class="col-md-12 text-center">
+      <div class="col-md-12 text-center" id="graffitiwallContainer">
         <canvas id="graffitiwall" width="1000" height="1050" style="image-rendering:pixelated;"></canvas>
+        <div id="nsfwWarning" class="mt-5 d-flex flex-column align-items-center position-absolute">
+          <h4>Warning: This canvas is unmoderated and may contain NSFW content. Do you want to continue?</h4>
+          <div class="btn btn-primary btn-sm d-none d-md-flex mt-2" onclick="acceptNSFW()">Accept</div>
```

Comment:
This button uses the wrong font color on darkmode:
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/114066135/348e0d9e-d7db-4be7-8b19-c1f901fa9d9a)
Please compare this with, for example, the view more buttons on the landing page:
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/114066135/534cec52-e5aa-4620-9782-697cc79d8ab7)

Reply:
should be correct now